CMakeLists.txt

```diff
@@ -45,7 +49,10 @@ set(CMAKE_MACOSX_RPATH 1)
 
 if(${PYTHON_VERSION_STRING} GREATER 3.0)
   message(STATUS "Using Python3")
-  find_package(Boost COMPONENTS python3 REQUIRED)
+  find_package(Boost COMPONENTS python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR})
+  if (NOT Boost_FOUND)
+    find_package(Boost COMPONENTS python3 REQUIRED)
+  endif()
```

Comment:
@jslee02 did you just come up with this snippet to find the Boost Python component, or is it from somewhere else?  It seems pretty arbitrary...

Reply:
I can't find the original snippet now, but it's from several repositories.
 Here are some of them:
* https://github.com/halide/Halide/blob/master/python_bindings/CMakeLists.txt#L7-L27
* https://pypkg.com/pypi/doom-py/f/doom_py/CMakeLists.txt
* https://sourceforge.net/p/netgen-mesher/git/ci/b3451ed2e473f7308f6cec6b0f9f058183e4aaba/

Yeah, I was surprised at this pattern too. :disappointed: